File: problem22-fibb/fibbonaci.js
/**
 * @name fib
 * @desc A function that takes an integer and returns the sequence of that fibbonaci
 * @return int
 */
function fib(n) {
  let a = 0;
  let b = 1;
  if (n < 0) {
    return a;
  } else if (n == 1) {
    return b;
  }
  for (let i = 2; i <= n; i++) {
    let c = a + b;
    a = b;
    b = c;
  }
  return b;
}

File: problem22-fibb/fibbonaci.py
'''
@name fib 
@desc A function that takes an integer and returns the sequence of the int 
@return int 
'''
def fib(n):
    a = 0
    b = 1
    if n < 0:
        return a
    elif n == 1:
        return b
    for i in range(2,n+1):
        c = a + b 
        a = b
        b = c
    return b
print(fib(10))